src/pages/RepoPage/SettingsTab/tabs/GeneralTab/Tokens/RepoUploadToken/RepoUploadToken.jsx

```diff
@@ -15,25 +14,13 @@ const TokenFormatEnum = Object.freeze({
 })
 
 function useRegenerateToken() {
-  const addToast = useAddNotification()
-  const { mutate, ...rest } = useRegenerateRepoUploadToken()
-
-  async function regenerateToken() {
-    mutate(null, {
-      onError: () =>
-        addToast({
-          type: 'error',
-          text: 'Something went wrong',
-        }),
-    })
-  }
-
-  return { regenerateToken, ...rest }
+  const { mutate, isLoading } = useRegenerateRepoUploadToken()
+  return { mutate, isLoading }
 }
 
 function RepoUploadToken({ uploadToken }) {
   const [showModal, setShowModal] = useState(false)
-  const { regenerateToken, isLoading } = useRegenerateToken()
+  const { mutate, isLoading } = useRegenerateToken()
```

Comment:
```suggestion
  const { mutate, isLoading } = useRegenerateRepoUploadToken()
```

And remove function on line 16.